static/css/inbox.less

```diff
@@ -1252,9 +1252,10 @@ a.fa:hover {
     .reset-filter {
       padding-left: 10px;
     }
-    .sort-results,
     .sort-dropdown {
-      float: right;
+      position: absolute;
+      right: 0;
+      margin-right: 10px;
```

Comment:
The icon was disappearing on mobile altogether. This fixes that and aligns it to the RHS.